100-level-AsyncAPI-Intro/step1.md

```diff
@@ -0,0 +1,54 @@
+Eve assumes her problem-solving role and approaches the spaceship. Inside, Eve discovers Chan, a programming genius visiting from Brownieterra. It becomes apparent that a technical glitch has affected the ship's communication system.
+
+With her expertise in event-driven architecture (EDA) and AsyncAPI, Eve swiftly springs into action. She formulates a plan to establish seamless communication among the spaceship's components in collaboration with Chan.
+
+Eve begins by providing Chan with a comprehensive explanation of event-driven architecture (EDA) and outlines the process of designing an EDA system using AsyncAPI.
+
+## Event-driven architecture
+
+An event signifies a change in the system's state that triggers communication between different services. Event-driven architecture (EDA) decomposes an application into several components interacting via events. These components encompass event producers and consumers:
+
+- An event **producer** identifies an event and transmits relevant information through the event channel.
+- An event **consumer** subscribes to an event channel to receive and process event data.
+
+Each component can fulfill the role of an event producer, consumer, or both.
+
+As Eve continues her conversation with Chan, she realizes she hasn't offered him a beverage. She quickly rushes to prepare a cup of coffee. While making the coffee, she draws upon a coffee shop as an analogy: a customer (producer) places an order for an espresso (event), conveying the order details to the barista (consumer). The sequence unfolds as follows:
+
+1. The customer initiates an espresso order (an event), and the barista receives the order details.
```

Comment:
Just wondering if it would make sense to add the role of the barista here as well? 

```suggestion
1. The customer initiates an espresso order (an event), and the barista (the event consumer) receives the order details.
```

Reply:
It definitely does, i would add customer as event producer as well.